python/phylanx/ast/physl.py

```diff
@@ -103,6 +103,9 @@ def __init__(self, func, tree, kwargs):
                 self.numpy_aliases.add(key)
         self.file_name = self.fglobals['__file__']
 
+        self.performance = not (kwargs.get('performance') is None)
```

Comment:
This should also work since None and False behave the same as a flag:
```suggestion
        self.performance = kwargs.get('performance')
```

Reply:
I did that before, but then this didn't work as I expected: https://github.com/STEllAR-GROUP/phylanx/pull/627/files#diff-2d3509625b4e7c1f8fb2ffdf8f3e53d2R171. Not sure why.